odoo80/Dockerfile

```diff
@@ -25,12 +23,10 @@ RUN apt-get update -q && apt-get upgrade -q \
     postgresql-common \
     libpq-dev \
     libldap2-dev \
-    libsasl2-dev \
     libxml2-dev \
     libxslt1-dev  \
     xsltproc \
     xmlstarlet \
-    openssl \
     python-libxml2 \
     phantomjs \
     bash-completion
```

Comment:
@moylop260 no, because I do a single clean up at [the end](https://github.com/vauxoo-dev/docker-odoo-image/blob/master-psql95-truiz/odoo80/Dockerfile#L54)